opendatahub/quickstart/common_docs/modelmesh-install.md

```diff
@@ -1,26 +1,45 @@
-# Install OpenDataHub ModelServing
+# Installing OpenDataHub ModelServing
+
+## Prerequisites
+
+- You have `git` installed.
+
+## Procedure
+
+1. Get the latest OpenDataHub ModelServing release:
 
-## Get the latest release
 ~~~
 git clone https://github.com/opendatahub-io/modelmesh-serving.git
+
 cd modelmesh-serving
 ~~~
 
-## Run install script
+2. Run the install script:
+
 ~~~
 source ../env.sh
+
 cd opendatahub/quickstart/basic
```

Comment:
This folder can be changeable so I think changing it to this would be better. What do you think?
```
cd opendatahub/quickstart/$QUICK_START_FOLDER   # (ex) cd opendatahub/quickstart/basic  
```